### PR TITLE
add splash page + basic styling

### PR DESCRIPTION
**Pull request by Bhammy**
Lots of changes because I waited too long to commit.

- splash component - very basic component for the splash page. basically some text and the login button.

- splash stylesheet - styling for the splash page, including big splashy background image. header is transparent. Suggesting a blue/sapphire theme but totally open to change.

- app.jsx - simplified the external routes; there is no more /signin or /welcome ; this should be accomplished through the splash page and button. Splash component is an Auth route.

### CODE REVIEW

File: .jshintrc [hunk 1]
```diff
@@ -0,0 +1 @@
+{ "esversion": 6 }
```
Comment (matthaws) [inline .jshintrc:1]: Add this to your .gitignore. If Ian and I are not using jshint, we don't need this file.

File: public/index.html [hunk 2]
```diff
@@ -10,6 +10,7 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="stylesheet" href='./index.css'>
```
Comment (matthaws) [inline public/index.html:13]: Remove stylesheet link from html. We're importing stylesheets in each related component.

File: src/components/splash/splash.css [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+p.eurovision {
+  height: 80px;
+  font-size: 80px;
+}
+
+.songcontest {
+  font-size: 30px;
+}
```
Comment (matthaws) [inline src/components/splash/splash.css:58]: A big part of our conversation moving forward will need to be if we are going to universally adopt BEM or not. I think we need to either all use it or none of us use it.

File: src/util/route_util.js [hunk 4]
```diff
@@ -17,7 +17,7 @@ const Protected = ({ component: Component, path, loggedIn }) => (
      loggedIn ? (
       <Component {...props} />
     ) : (
-      <Redirect to="/signin" />
+      <Redirect to="/" />
```
Comment (matthaws) [inline src/util/route_util.js:20]: Both AuthRoute and ProtectedRoute redirect to the same page?

**Review by matthaws — CHANGES_REQUESTED** [review]
See notes in the "files changed" tab.

File: src/components/splash/splash.css (hunk 3 above)
Comment (Bhammy) [reply src/components/splash/splash.css:58]: Agreed! Waiting for input on this - I wanted to put something up as a proof-of-concept but 100% not married to it.

File: src/util/route_util.js (hunk 4 above)
Comment (Bhammy) [reply src/util/route_util.js:20]: Waiting to see what our logged-in landing page is - I've put in /home for now.

**Review by matthaws — APPROVED** [review]

**Pull request merged**